### DISCUSSION TITLE
"macrecovery.py" is not recognised in windows

### DISCUSSION THREAD

**Discussion by nandup4134** [cheneyveron/clover-x79-e5-2670-gtx650#81] [Q&A] [answered]
When I am creating hackintosh mac os Big Sur Internet recovery usb bootable this error (“**macrecovery.py** is not recognized as an internal or external command”) arises in cmd. Please tell how to solve this

This is my first post in GitHub and also first time to install **Hackintosh**, sorry for any errors

**Comment by ghost** [comment, marked as answer]
Have you installed Python to your system? 
It comes preinstalled with macOS.

**Reply by cheneyveron** [reply]
FYI, python can be downloaded [here](https://www.python.org/downloads/). And I recommand to tick 'Add to PATH' in installer.
